src/provider/application.js

```diff
@@ -141,9 +141,9 @@ class Application extends EventEmitter {
   */
   launch() {
     if (window.self !== window.top) {
-      // 1: Setup listeners for all incoming communication and beforeunload
+      // 1: Setup listeners for all incoming communication and unload
       window.addEventListener('message', this.handleConsumerMessage);
-      window.addEventListener('beforeunload', this.unload);
+      window.addEventListener('unload', this.unload);
```

Comment:
`beforeunload` - This event fires when a window is about to unload its resources. so href with tel: was triggering this event which is about to unload, but it shouldn't ideally, because its not really unloading a page(or opening a new page)
`unload` event occurs once a page has unloaded, so this is stopping from href="tel:..." to trigger unload